### ISSUE TITLE
Use a proper logger on the frontend & backend

### ISSUE THREAD

**Issue by nikhilxb** [vizstack/vizstack#61]
Right now, `console.debug` is not good enough because it doesn't give me options to log what I want. Do we even have logging on the backend?

**Comment by ryanholmdahl** [comment]
This seems like it was originally intended for vz-atom, where a single program had a unified frontend and backend. That said, it'd be really cool if all of our projects used vz-logger! I'm not totally sure if that works given the circular dependency, but I'll leave this open to consider it in the future.

**Comment by nikhilxb** [comment]
Yes that would be cool and we should see if that works. I imagine having a devDependency of the published vz-logger would fine
